src/Sarif/Microsoft.Json.Schema.ToDotNet.targets

```diff
@@ -21,7 +21,7 @@
       <FileWrites Include="$(_ObjectModelOutputDirectory)" />
     </ItemGroup>
 
-    <Exec Command="&quot;$(JsonSchemaToDotNetPath)&quot; --schema-file-path &quot;%(JsonSchemaFile.FullPath)&quot; --output-directory &quot;$(_ObjectModelOutputDirectory)&quot; --force-overwrite --namespace-name %(JsonSchemaFile.Namespace) --root-class-name %(JsonSchemaFile.RootClassName) --copyright-file-path &quot;%(JsonSchemaFile.CopyrightFilePath)&quot; --hints-file-path &quot;%(JsonSchemaFile.HintsFilePath)&quot;" />
+    <Exec Command="&quot;$(JsonSchemaToDotNetPath)&quot; --schema-name Sarif --schema-file-path &quot;%(JsonSchemaFile.FullPath)&quot; --output-directory &quot;$(_ObjectModelOutputDirectory)&quot; --force-overwrite --namespace-name %(JsonSchemaFile.Namespace) --root-class-name %(JsonSchemaFile.RootClassName) --copyright-file-path &quot;%(JsonSchemaFile.CopyrightFilePath)&quot; --hints-file-path &quot;%(JsonSchemaFile.HintsFilePath)&quot;" />
```

Comment:
Version `0.7.0` also adds the `--schema-name` command line option. This name forms a part of some of the generated type names, in particular `ISarifNode` and `SarifRewritingVisitor`. I could have used the first component of the schema file name `Sarif.json.schema` but I didn't want to rely on the file having a particular name.